app/components/security/file-actions/index.js

```diff
@@ -53,6 +53,19 @@ export default class FileActionsNewComponent extends Component {
     return [ireneHost, 'file', fileId].join('/');
   }
 
+  manualScanStatus(scanStatus) {
```

Comment:
rename this to `manualScanStatusText` and `scanStatus` can directly destructured like this `manualScanStatusText([scanStatus])`